### PR TITLE
Add links for DXF 

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs/pull/17522 https://github.com/pingcap/docs/pull/17637
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-add-index.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-add-index/','/docs-cn/dev/r
 
 `ALTER TABLE.. ADD INDEX` 语句用于在已有表中添加一个索引。在 TiDB 中，`ADD INDEX` 为在线操作，不会阻塞表中的数据读写。
 
+> **Tip:**
+>
+> 你可以使用 [TiDB 分布式执行框架](/tidb-distributed-execution-framework.md)加速执行该语句。
```
Comment (qiancai) [inline sql-statements/sql-statement-add-index.md:13]:
```suggestion
> 你可以使用 [TiDB 分布式执行框架](/tidb-distributed-execution-framework.md)加速该语句的执行。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: tidb-distributed-execution-framework.md [hunk 2]
```diff
@@ -23,29 +23,30 @@ TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩
...
 ## 使用限制
 
-分布式执行框架最多同时调度 16 个任务（包括 `ADD INDEX` 和 `IMPORT INTO`）。
+分布式执行框架最多同时调度 16 个任务（包括 [`ADD INDEX`](/sql-statements/sql-statement-add-index.md) 和 [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md)）。
 
 ### `ADD INDEX` 的使用限制
 
-- 集群内同时只能有一个 `ADD INDEX` 任务进行分布式执行。如果在当前的 `ADD INDEX` 分布式任务还未执行完成时就提交了一个新的 `ADD INDEX` 任务，则新提交的 `ADD INDEX` 任务不会被该框架调度，而是直接通过事务的方式来执行。
+- 在 v8.1.0 之前，集群内同时只能有一个 [`ADD INDEX`](/sql-statements/sql-statement-add-index.md) 任务进行分布式执行。如果在当前的 [`ADD INDEX`](/sql-statements/sql-statement-add-index.md) 分布式任务还未执行完成时就提交了一个新的 [`ADD INDEX`](/sql-statements/sql-statement-add-index.md) 任务，则新提交的 [`ADD INDEX`](/sql-statements/sql-statement-add-index.md) 任务不会被该框架调度，而是直接通过事务的方式来执行。
+- 从 v8.1.0 开始，分布式执行框架支持并行执行多个 [`ADD INDEX`](/sql-statements/sql-statement-add-index.md) 任务。
```
Comment (qiancai) [inline tidb-distributed-execution-framework.md:44]: 这是 PM 要求加的吗

Comment (hfxsd) [reply tidb-distributed-execution-framework.md:44]: @Frank945946 PTAL

Comment (hfxsd) [inline tidb-distributed-execution-framework.md:44]:
```suggestion
```

**Review by Frank945946 — APPROVED** [review]

**Pull request merged**